supervisor/api/addons.py

```diff
@@ -250,7 +250,7 @@ async def info(self, request: web.Request) -> dict[str, Any]:
             ATTR_WEBUI: addon.webui,
             ATTR_INGRESS_ENTRY: addon.ingress_entry,
             ATTR_INGRESS_URL: addon.ingress_url,
-            ATTR_INGRESS_PORT: addon.ingress_port,
+            ATTR_INGRESS_PORT: await addon.get_ingress_port(),
```

Comment:
maybe we can call that once on install and afterward we just use the property with the value from cache

Reply:
What I mean, we register the port once and the add-on use over the full live circle the same port. Means we only need call this async once we install the add-on (or update maybe as well) afterwards it's just a simple property from cached data in memory

Reply:
Having `check_port` async is fine, just that part with ingress port handling of the PR should get handle better